blockchain/pkg/client.go

```diff
@@ -0,0 +1,31 @@
+package pkg
+
+import (
+	gsrpc "github.com/centrifuge/go-substrate-rpc-client/v4"
+
+	"github.com/cerebellum-network/cere-ddc-sdk-go/blockchain/pkg/pallets"
+)
+
+type Client struct {
+	*gsrpc.SubstrateAPI
+
+	DDCClusters  *pallets.DdcClustersApi
```

Comment:
lets do lower case for attribute as well

Reply:
Sorry, updated.